src/ssh/azext_ssh/_help.py

```diff
@@ -20,6 +20,7 @@
         - name: Give the public IP (or hostname) of a VM to SSH to
           text: |
             az ssh vm --ip 1.2.3.4
+            az ssh vm --hostname example.com
```

Comment:
We need to add a new example name for this new text

Reply:
what do you mean?  example.com is a valid IETF name

Reply:
I believe @zhoxing-ms means you may give it a new name, like L20.

Reply:
they are an alias of the same argument so I don't see why we would do so.  e.g. you can give IP to --hostname and a hostname to --ip .. I just added the `--hostname` alias.